不支持 node 环境打包

### System Info

Cannot find module 'node:path'

### Details

Cannot find module 'node:path'

### Reproduce link

_No response_

### Reproduce Steps

pnpm i

--- a/examples/express/package.json
+++ b/examples/express/package.json
@@ -1,0 +1,22 @@
+{
+  "name": "example-express",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "dev": "rspack serve",
+    "build": "rspack build"
+  },
+  "devDependencies": {
+    "@rspack/cli": "workspace:*",
+    "run-script-webpack-plugin": ""
+  },
+  "dependencies": {
+    "express": "4.18.2"
+  },
+  "sideEffects": false,
+  "keywords": [],
+  "author": "",
+  "license": "MIT"
+}

--- a/examples/express/rspack.config.js
+++ b/examples/express/rspack.config.js
@@ -1,0 +1,50 @@
+const { RunScriptWebpackPlugin } = require("run-script-webpack-plugin");
+
+/** @type {import('@rspack/cli').Configuration} */
+const config = {
+	experiments: {
+		rspackFuture: {
+			disableTransformByDefault: true
+		}
+	},
+	context: __dirname,
+	target: "node",
+	entry: {
+		main: ["webpack/hot/poll?100", "./src/main.ts"]
+	},
+	module: {
+		rules: [
+			{
+				test: /\.ts$/,
+				use: {
+					loader: "builtin:swc-loader",
+					options: {
+						jsc: {
+							parser: {
+								syntax: "typescript",
+								decorators: true
+							}
+						}
+					}
+				}
+			}
+		]
+	},
+	optimization: {
+		minimize: false
+	},
+	externalsType: "commonjs",
+	plugins: [
+		!process.env.BUILD &&
+			new RunScriptWebpackPlugin({
+				name: "main.js",
+				autoRestart: false
+			})
+	].filter(Boolean),
+	devServer: {
+		devMiddleware: {
+			writeToDisk: true
+		}
+	}
+};
+module.exports = config;

--- a/examples/express/src/api/root.ts
+++ b/examples/express/src/api/root.ts
@@ -1,0 +1,3 @@
+export function rootHandler(req, res) {
+	res.send("Hello Rspack");
+}

--- a/examples/express/src/main.ts
+++ b/examples/express/src/main.ts
@@ -1,0 +1,15 @@
+declare const module: any;
+import express from "express";
+import { rootHandler } from "./api/root";
+async function bootstrap() {
+	const app = express();
+	app.get("/", rootHandler);
+	const server = app.listen(4000, () =>
+		console.log("Listening on http://localhost:4000")
+	);
+	if (module.hot) {
+		module.hot.accept();
+		module.hot.dispose(() => server.close());
+	}
+}
+bootstrap();

--- a/packages/rspack/src/builtin-loader/swc/relay.ts
+++ b/packages/rspack/src/builtin-loader/swc/relay.ts
@@ -1,4 +1,4 @@
-import path from "node:path";
+import path from "path";
 
 import { RawRelayConfig } from "@rspack/binding";
 

